### PR TITLE
Simplify get started web sample

### LINKED ISSUE #3441: Simplify login web get started code
Simplifying the code will also require updating the codelab.

@johnpryan

### PR DESCRIPTION
**Pull request by johnpryan**
@sfshaza2 this is ready for a review. I might not have included all the necessary changes to the text, but this has the starter code and completed sample included in the markdown

fixes #3441

### CODE REVIEW

**Comment by sfshaza2** [comment]
Staging this to take a close look.

File: src/docs/get-started/codelab-web.md [hunk 1]
```diff
@@ -301,40 +393,52 @@ and update the app’s UI when the form is complete.
...
+  That is something you can add later using form validation, if you like.
 {{site.alert.end}}
 
 <ol markdown="1">
-<li markdown="1">Add a `_formCompleted` field.<br>
-At the top of the `_SignUpFormState` class, add a boolean
-field that you'll set to true when all three fields contain text.
+<li markdown="1">Add an _updateFormProgress method
+In the `_SignUpFormState` class, after the fields are defined add a new
+method called _updateFormProgress():
```
Comment (sfshaza2) [inline src/docs/get-started/codelab-web.md]: Rephrase this sentence, as it doesn't make sense currently with Add "_updateFormProgress twice". Also, codefont `_updateFormProgress`.

File: src/docs/get-started/codelab-web.md [hunk 2]
```diff
@@ -133,14 +136,96 @@ and the web server when you want to test on other browsers.
 
 <li markdown="1">The starting app is displayed in the following DartPad.
```
Comment (sfshaza2) [inline src/docs/get-started/codelab-web.md]: There's a missing </li> before this line.  I believe that is resulting in the errant `</ol>` in the [generated page at the end of step 2](https://sz-flutter.firebaseapp.com/docs/get-started/codelab-web#step-2-enable-sign-in-progress-tracking).

File: src/docs/get-started/codelab-web.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 title: Write your first Flutter app on the web
```
Comment (sfshaza2) [inline src/docs/get-started/codelab-web.md]: Towards the end of the doc there's a visible "[todo: replace screenshot]". Do you think you could do that? Please?  :)

File: src/docs/get-started/codelab-web.md [hunk 4]
```diff
@@ -572,158 +678,283 @@ The animation has the following behavior:
...
+}
 ```
 </li>
 
-<li markdown="1">Create a `Tween` sequence.<br>
-Also in `initState()`, define a `Tween` sequence consisting
-of three color tweens, one for each color of the animation
-(red-to-orange, orange-to-yellow, yellow-to-green).
+Then, replace the `LinearProgressIndicator` in the Form with this new
+`AnimatedProgressIndicator`:
```
Comment (sfshaza2) [inline src/docs/get-started/codelab-web.md]: Scroll to the end of [step 3 in the generated page](https://sz-flutter.firebaseapp.com/docs/get-started/codelab-web#step-3-add-animation-for-sign-in-progress). The code following the codeblock should be in its own codeblock.

File: src/docs/get-started/codelab-web.md [hunk 5]
```diff
@@ -262,6 +353,7 @@ Click it to bring up the welcome screen.
 Note how it animates in from the bottom.
 You get that behavior for free.
 </li>
+
 </ol>
 
 ### Problems?
```
Comment (sfshaza2) [inline src/docs/get-started/codelab-web.md]: Correct me if I'm wrong, but now that we have the DartPad at the end with the complete example, why not just delete the three links to the gist files?

File: src/docs/get-started/codelab-web.md (hunk 1 above)
Comment (johnpryan) [reply src/docs/get-started/codelab-web.md]: fixed!

File: src/docs/get-started/codelab-web.md (hunk 2 above)
Comment (johnpryan) [reply src/docs/get-started/codelab-web.md]: fixed

File: src/docs/get-started/codelab-web.md (hunk 3 above)
Comment (johnpryan) [reply src/docs/get-started/codelab-web.md]: fixed, I updated the other DevTools images to use an empty chrome shell

File: src/docs/get-started/codelab-web.md (hunk 4 above)
Comment (johnpryan) [reply src/docs/get-started/codelab-web.md]: fixed

File: src/docs/get-started/codelab-web.md (hunk 5 above)
Comment (johnpryan) [reply src/docs/get-started/codelab-web.md]: I decided to remove the problems sections altogether. WDYT?

**Comment by johnpryan** [comment]
@sfshaza2 ready for another look

**Comment by sfshaza2** [comment]
Will do later today. Sorry for the delay!

**Comment by sfshaza2** [comment]
Staged: https://sz-flutter-2.firebaseapp.com/docs/get-started/codelab-web

It looks vunderbar!!!

**Pull request merged**